Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -143,8 +153,9 @@ extension AutofillUserScript {
     }
 
     struct CredentialObject: Codable {
-        let id: Int64
+        let id: String
```

Comment:
I remember a while back we changed some of the data type casting we were doing for a good reason, but I don't remember what the reason was. Just want to check everyone on the JS side is aware of the change?

Reply:
@THISISDINOSAUR, good point! The reason we need to change the type here is that Bitwarden ids are Strings. @GioSensation informed me, JS layer already worked with Strings even before this change.

Reply:
Oh interesting, I assume it's fine then. tbh I don't remember what the original change was, @shakyShane do you remember?